### PR TITLE
created workers, lb and sg

### PR DESCRIPTION
**Pull request by mustafa-bungalow**

### CODE REVIEW

**Comment by mustafa-bungalow** [comment]
The plan is so big.... run locally

File: terraform/variables.tf [hunk 1]
```diff
@@ -0,0 +1,24 @@
+variable "region" {
+  type    = string
+  default = "us-west-2"
+}
+
+variable "project" {
+  type    = string
+  default = "it"
```
Comment (ableeck) [inline terraform/variables.tf]:
```suggestion
  default = "sandbox"
```

File: terraform/variables.tf [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+variable "resource_group" {
+  type    = string
+  default = "network"
```
Comment (ableeck) [inline terraform/variables.tf]:
```suggestion
  default = "sandbox"
```

File: terraform/variables.tf [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+variable "repo_name" {
+  type    = string
+  default = "bungalow-it"
```
Comment (ableeck) [inline terraform/variables.tf]:
```suggestion
  default = "bungalow-devops-hiring-public"
```

File: terraform/sg.tf [hunk 4]
```diff
@@ -0,0 +1,53 @@
+resource "aws_security_group" "sg" {
+  name        = "${local.prefix}-sg"
+  description = "Security group usage with ${local.worker_1_ec2}-instance"
+  vpc_id      = module.main-vpc.vpc_id
+
+  tags = {
+    Name = "${local.worker_1_ec2}-sg"
```
Comment (ableeck) [inline terraform/sg.tf]:
```suggestion
    Name = "${local.prefix}-sg"
```

File: randomnumber/randomnumber/models.py [hunk 5]
```diff
@@ -6,7 +6,7 @@
 
 class RandomNumber(models.Model):
     def get_random_number() -> int:
-        return random.getrandbits(64)
+        return random.getrandbits(random.randint(60, 64))
```
Comment (ktrain) [inline randomnumber/randomnumber/models.py:9]: This is to make overflows happen more often, right?

Comment (ableeck) [reply randomnumber/randomnumber/models.py:9]: Less often

**Review by ableeck — APPROVED** [review]

**Pull request merged**